README.md

```diff
@@ -344,6 +344,16 @@ Note that, according to the doc, the inherent version of the address is a semant
 
 To help you write markdown descriptions in Go, a simple builder is available in the sub-package `markdown`. This is quite handy to avoid conflicts with backticks that are both used in Go for litteral multi-lines strings and code blocks in markdown.
 
+#### Providing Examples for Structs
```

Comment:
"for Structs" should be replaced by "custom types". They don't necesseraly have a `Struct` kind.